src/api/app/views/webui2/webui/users/tasks/index.html.haml

```diff
@@ -0,0 +1,62 @@
+              'data-toggle': 'tab', 'aria-controls': 'requests-in-tab', 'aria-selected': 'true', role: 'tab' }
+              Incoming Requests (#{User.current.incoming_requests.count})
+        %li.nav-item
+          %a.nav-link.text-nowrap#requests-out-tab{ href: '#requests-out', title: "Requests that #{User.current} has sent",
+            'data-toggle': 'tab', 'aria-controls': 'requests-out-tab', 'aria-selected': 'false', role: 'tab' }
+            Outgoing Requests (#{User.current.outgoing_requests.count})
+        %li.nav-item
+          %a.nav-link.text-nowrap#requests-declined-tab{ href: '#requests-declined', title: "Requests from #{User.current} that are declined",
+            'data-toggle': 'tab', 'aria-controls': 'requests-declined-tab', 'aria-selected': 'false', role: 'tab' }
+            Declined Requests (#{User.current.declined_requests.count})
+        %li.nav-item
+          %a.nav-link.text-nowrap#all-requests-tab{ href: '#all-requests', title: "All Requests from #{User.current}",
+            'data-toggle': 'tab', 'aria-controls': 'all-requests-tab', 'aria-selected': 'false', role: 'tab' }
+            All Requests (#{User.current.requests.count})
+        %li.nav-item.dropdown
+          %a.nav-link.dropdown-toggle{ href: '#', 'data-toggle': 'dropdown', 'role': 'button', 'aria-expanded': 'false', 'aria-haspopup': 'true' }
+          .dropdown-menu.dropdown-menu-right
+    .card-body
+      .tab-content
+        .tab-pane.fade.show.active#requests-in{ role: 'tabpanel', 'aria-labelledby': 'requests-in-tab' }
+          = render(partial: 'webui2/shared/requests_table', locals: { id: 'requests_in_table', source_url: user_requests_path(User.current) })
+        .tab-pane.fade#requests-out{ role: 'tabpanel', 'aria-labelledby': 'requests-out-tab' }
+          = render(partial: 'webui2/shared/requests_table', locals: { id: 'requests_out_table', source_url: user_requests_path(User.current) })
+        .tab-pane.fade#requests-declined{ role: 'tabpanel', 'aria-labelledby': 'requests-declined-tab' }
+          = render(partial: 'webui2/shared/requests_table', locals: { id: 'requests_declined_table', source_url: user_requests_path(User.current) })
+        .tab-pane.fade#all-requests{ role: 'tabpanel', 'aria-labelledby': 'all-requests-tab' }
+          = render(partial: 'webui2/shared/requests_table', locals: { id: 'all_requests_table', source_url: user_requests_path(User.current) })
+
+  - involved_patchinfos = User.current.involved_patchinfos
+  - if involved_patchinfos.present?
+    .card.mt-3#patchinfos
+      .bg-light
+        %ul.nav.nav-tabs.pt-2.px-3.flex-nowrap.collapsible{ 'role': 'tablist' }
+          %li.nav-item
+            %a.nav-link.text-nowrap.active{ href: '#patchinfos-in', title: "Requests that #{User.current} has to merge" }
+              Maintenance Requests (#{involved_patchinfos.count})
```

Comment:
`involved_patchinfos` is an array. Do not use `count` with arrays. :see_no_evil: 

Compare implementation of `count` and `lenght/size`:

https://apidock.com/ruby/Array/count
https://apidock.com/ruby/v1_9_3_392/Array/length

actually, in general when not being sure what kind of object this is, use size. :wink: